delfi_attributes.json

```diff
@@ -0,0 +1,1022 @@
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2092,
+	  "LABEL": "Grundfläche des Aufzugs",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {
+		"width": [],
+		"length": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2093,
+	  "LABEL": "Grundflächenlänge",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {
+		"length": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2094,
+	  "LABEL": "Grundflächenbreite",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {
+		"width": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2095,
+	  "LABEL": "Verbindungsfunktion",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2100,
+	  "LABEL": "Stufe vorhanden",
+	  "CONDITION": [
+		{
+		  "osm_tags": {
+			"highway": "steps",
+			"step_count": "1"
+		  }
+		},
+		{
+		  "osm_tags": {
+			"barrier": "kerb",
+			"kerb": [
+			  "lowered",
+			  "flush",
+			  "raised",
+			  "no"
+			]
+		  }
+		}
+	  ],
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2101,
+	  "LABEL": "Stufenhöhe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "step_count": "1"
+		}
+	  },
+	  "TAGS": {
+		"step:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2101,
+	  "LABEL": "Stufenhöhe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "barrier": "kerb",
+		  "kerb": [
+			"lowered",
+			"raised"
+		  ]
+		}
+	  },
+	  "TAGS": {
+		"kerb:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2110,
+	  "LABEL": "Treppe vorhanden",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2111,
+	  "LABEL": "Verbindungsfunktion der Treppe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2112,
+	  "LABEL": "Stufenhöhe (Tritthöhe)",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"step:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2113,
+	  "LABEL": "Anzahl der Treppenstufen",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"step_count": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2120,
+	  "LABEL": "Rampe und Wege mit Neigungen vorhanden/Zugang über Rampe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": [
+			"footway",
+			"path"
+		  ],
+		  "incline": true
+		}
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2121,
+	  "LABEL": "Verbindungsfunktion",
+	  "CONDITION": {
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2123,
+	  "LABEL": "Rampenbreite",
+	  "CONDITION": {
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"width": []
+	  }
+	},
```

Comment:
Please limit the condition for both DELFI attributes to ramps using:


```
		"osm_tags": {
		  "highway": [
			"footway",
			"path"
		  ],
		  "incline": true
		}
```


Also, even though it does make little sense, I would still add the following for completeness:

```


	{
	  "DELFI_ID": 2124,
	  "LABEL": "Rampenneigung ",
	  "CONDITION": {
		"osm_tags": {
		  "highway": [
			"footway",
			"path"
		  ],
		  "incline": true
		}
	  },
	  "TAGS": {
                "incline": []
          }
	},
```